addons/sale/static/src/js/product_configurator_dialog/product_configurator_dialog.xml

```diff
@@ -2,11 +2,12 @@
 <templates xml:space="preserve">
     <t t-name="sale.ProductConfiguratorDialog">
         <Dialog size="size" title="title" contentClass="'o_sale_product_configurator_dialog'">
-            <ProductList t-if="this.state.products.length" products="this.state.products"/>
+            <ProductList t-if="this.state.products.length" products="this.state.products" options="this.props.options"/>
```

Comment:
As mentioned above, no need for these options here or below, you can directly get `showQuantity` and `showPrice` from the env.